corehq/apps/cloudcare/static/cloudcare/js/formplayer/constants.js

```diff
@@ -14,6 +14,8 @@ hqDefine("cloudcare/js/formplayer/constants", function () {
             GRID: 'grid',
             LIST: 'list',
         },
+        ALLOWED_FIELD_ALIGNMENTS: ['start', 'end', 'center', 'left', 'right'],
```

Comment:
do we really need start and end? Aren't they just synonyms for left and right?

Reply:
Yes, if we're allowing vertical alignment as well, since 'start' and 'end' are the css values for top or bottom vertical alignment. I could break them into separate lists to be more explicit (and since 'left' and 'right' have no bearing on vertical alignment) -- this would look like`['start', 'center', 'end']` and `['left', 'center', 'right']` -- but it seemed cleaner to me to do it this way.